pkg/bb/cmd/main.go

```diff
@@ -25,8 +25,21 @@ func main() {
 
 func init() {
 	m := func() {
-		if len(os.Args) <= 1 {
-			log.Fatalf("You need to specify which command to invoke.")
+		if len(os.Args) == 0 {
+			log.Fatal("Arg len is 0. This is impossible")
+		}
+		if len(os.Args) == 1 {
+			// This might be a symlink, and have been invoked by an sshd.
```

Comment:
Can you make `init` also use `bin/sh` and get rid of `bin/defaultsh` altogether?

Reply:
Unfortunately, this solution works for bb, but not for source mode.

Reply:
yeah. What should we do? Are you ok with this in source mode too?